app/views/pipeline_outputs/show.html.erb

```diff
@@ -26,19 +26,29 @@
 </p>
 
 <p>
-  <strong>Report(s):</strong>
-  <% @pipeline_output.reports[0...-1].each do |r| %>
-    <%= link_to r.name, r if r %>, 
+  <% if @pipeline_output.reports.count == 0 %>
```

Comment:
Similiar to above, you can simplify this.

Reply:
ok